src/App.js

```diff
@@ -97,6 +97,8 @@ import { SignMessageRequestScreen } from './screens/Reown/SignMessageRequestScre
 import { SignOracleDataRequestScreen } from './screens/Reown/SignOracleDataRequestScreen';
 import { CreateTokenRequestScreen } from './screens/Reown/CreateTokenScreen';
 import { SuccessFeedbackScreen } from './screens/Reown/SuccessFeedbackScreen';
+import UnifiedQRScanner from './screens/UnifiedQRScanner';
```

Comment:
Does it make sense to remove the screens that are not used anymore? (QRCode reader for token, nanos, reown. Manual register also - if not used)

Reply:
I don't think so, they're still available in the Settings screen.